src/main/java/com/massivecraft/factions/cmd/CmdTntFill.java

```diff
@@ -39,109 +42,106 @@ public void perform(CommandContext context) {
             return;
         }
 
-        context.msg(TL.COMMAND_TNTFILL_HEADER);
-        int radius = context.argAsInt(0, 16);
-        int amount = context.argAsInt(1, 16);
-        if (radius > SavageFactions.plugin.getConfig().getInt("Tntfill.max-radius")) {
-            context.msg(TL.COMMAND_TNTFILL_RADIUSMAX.toString().replace("{max}", SavageFactions.plugin.getConfig().getInt("Tntfill.max-radius") + ""));
-            return;
-        }
-        if (amount > SavageFactions.plugin.getConfig().getInt("Tntfill.max-amount")) {
-            context.msg(TL.COMMAND_TNTFILL_AMOUNTMAX.toString().replace("{max}", SavageFactions.plugin.getConfig().getInt("Tntfill.max-amount") + ""));
-            return;
-        }
-
+        // Don't do I/O unless necessary
```

Comment:
A configuration just gets loaded into a HashMap when the plugin enables (that's how Bukkit's API does it), so it's not necessarily I/O.